modules/lotamePanoramaIdSystem.js

```diff
@@ -78,17 +83,40 @@ function getFromStorage(key) {
     const storedValueExp = storage.getDataFromLocalStorage(
       `${key}_exp`, undefined
     );
-    if (storedValueExp === '') {
+
+    if (storedValueExp === '' || storedValueExp === null) {
       value = storage.getDataFromLocalStorage(key, undefined);
     } else if (storedValueExp) {
-      if ((new Date(storedValueExp)).getTime() - Date.now() > 0) {
+      if ((new Date(parseInt(storedValueExp, 10))).getTime() - Date.now() > 0) {
         value = storage.getDataFromLocalStorage(key, undefined);
       }
     }
   }
   return value;
 }
 
+/**
+ * Returns the domain used to host the ID request 
+ */
+function getIDHost(config) {
```

Comment:
What if instead, just have a url override? I added a potential regex check fo extra safety
`
let host = ID_HOST;
if (config && config['url'] && config['ur'l .match(/id\.\w+\.lotame\.com/) {
   host = config['url'];
}
return host;
`

Reply:
I know this comment is a week old, but for future reference, I pulled out the dynamic id endpoint determination. Our build script will find/replace the production URL with the appropriate one for the environment. That will keep this code simple and avoid having to pass some kind of value from the page, something that I wasn't sure the Prebid folks would accept anyway.